### DISCUSSION TITLE
Confusion about correct cdos258 floppy image, thn#037 has hdtest on the image but latest IMAGE from github doesn't

### DISCUSSION THREAD

**Discussion by riwide** [thehighnibble/thehighnibble.github.io#57] [Cromemco Z-1 replica]
The thn#037 video refers to cdos258 image, built with hd support and containing hdtest program.
Using the latest github sd image and udsing cdos258-8 I do not get the hdtest program and also I cannot select drive E if  HD0 is configured.

It seems as though the latest cdos floppy is not the correct one for hd support.

Where can I find the cdos258.dsk  with hd support?

**Comment by pfcushing** [comment]
After looking for HDTEST myself, I finally found that  it is available on the Github  Cromemco Vintage Computer Repository, which requires a membership per their rules and agreeing to stipulated terms which include not sharing anything obtained from that Github site.  HDTEST is a nice tool but probably only for troubleshooting HD issues and does not provide any capabilities other than that.
Paul Cushing

**Reply by riwide** [reply]
Thanks for the info.  I have managed without using HDTEST, but I will explore that later.
